shaq/_cli.py

```diff
@@ -164,11 +164,74 @@ def _parser() -> argparse.ArgumentParser:
         type=str,
         help="send the request to a proxy server",
     )
+    advanced_group.add_argument(
+        "--metadata",
+        action="store_true",
+        help="Overwrite titel and metadata of the song",
+    )
     return parser
 
+def updateMetadata(console, title, artist, path):
```

Comment:
Please match the style of the rest of this file -- this should be `update_metadata`.

(Also, please add type hints like the other functions -- this package is intentionally typed to make it harder for me to introduce bugs/unintentional API changes.)